web/src/main/kotlin/no/nav/su/se/bakover/web/routes/grunnlag/OppholdIUtlandetVilkårJson.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.su.se.bakover.web.routes.grunnlag
+
+import no.nav.su.se.bakover.domain.vilkår.OppholdIUtlandetVilkår
+import no.nav.su.se.bakover.domain.vilkår.Resultat
+import no.nav.su.se.bakover.service.vilkår.LeggTilOppholdIUtlandetRequest
+
+internal data class OppholdIUtlandetVilkårJson(
+    val vilkår: String,
+    val status: LeggTilOppholdIUtlandetRequest.Status,
```

Comment:
Dette var en rar import. Hvorfor bor Status i servicen? Statusen bør bo i domain (Og det gjør den vel allerede som Resultat), LeggTilOppholdIUtlandetRequest bør bo der den ligger, i service; og så kan vi enten ha en String i routes eller evt. en StatusJson hvis det er ønskelig med en sterkere kontrakt mellom frontend/backend.

Reply:
Havner vel litt i skvis her.. på den ene siden ønsker vi å bevege oss bort fra disse statusene (tilhørte tidligere behandlingsinformasjon) og over mot å la vilkårsvurderingene bestemme dette basert på grunnlagsdata. Inntil videre bruker vi fortsatt denne statusen i frontend (ska være i utlandet/skal oppholde seg i norge/uavklart) - har derfor flyttet den lenger ut slik at den enkelt kan fjernes når den tid kommer. Inntil da vil den brukes for å mappe frontend-verdiene (status) til aktuelt resultat (domene).